src/engine/renderer/glsl_source/ssao_fp.glsl

```diff
@@ -23,7 +23,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 /* ssao_fp.glsl */
 
 uniform sampler2D u_DepthMap;
-IN(flat) vec3 unprojectionParams;
```

Comment:
I guess the code in ssao_vp that sets this should be deleted?

Reply:
Yeah, must've forgoten to add it to the commit.

Reply:
Fixed.